sailfishusermanager.cpp

```diff
@@ -348,3 +370,54 @@ bool SailfishUserManager::hasAccessRights(uint uid_to_modify) {
 
     return true;
 }
+
+void SailfishUserManager::switchUser(uint uid)
```

Comment:
I think this should probably also check that uid is an existing uid.

Reply:
Check added